atomate2/openmm/jobs/core.py

```diff
@@ -0,0 +1,147 @@
+
+from typing import List, Union, Optional, Dict
+from pathlib import Path
+
+from dataclasses import dataclass
+
+from jobflow import Maker
+
+from pymatgen.io.openmm.schema import InputMoleculeSpec
+from pymatgen.io.openmm.sets import OpenMMSet
+
+from atomate2.openmm.jobs.base import BaseOpenmmMaker
+
+
+@dataclass
+class InputMaker(Maker):
+    name: str = "input maker"
+
+    def make(
+        self,
+        input_mol_dicts: List[Union[Dict, InputMoleculeSpec]],
+        density: Optional[float] = None,
+        box: Optional[List[float]] = None,
+        prev_dir: str | Path | None = None
+    ):
+        """
+        Create a job that generates an OpenMM input set.
+
+        Parameters
+        ----------
+        input_set : .OpenMMSet
+            A pymatgen structure object.
+        density : float
+        box : list
+        prev_dir : str or Path or None
+            A previous OpenMM simulation directory to link files from.
+
+        Returns
+        -------
+        Job
+            A OpenMM job containing one npt run.
+        """
+        return
+
+
+@dataclass
+class EnergyMinimizationMaker(BaseOpenmmMaker):
+    name: str = "energy minimization"
```

Comment:
Is `EnergyMinimization` used to optimize the coordinates of atoms before the simulation starts?